polaris-react/src/components/PolarisTestProvider/PolarisTestProvider.tsx

```diff
@@ -67,7 +67,7 @@ export function PolarisTestProvider({
 
   const featuresConfig = useMemo(
     () => ({
-      polarisSummerEditions2023: false,
+      polarisSummerEditions2023: true,
```

Comment:
Temp, we should remove this property in https://github.com/Shopify/polaris/issues/10189!